src/styles/components/header.scss

```diff
@@ -67,11 +67,87 @@
 	&__menu-icon {
 		display: flex;
 		justify-content: center;
+		cursor: pointer;
 
 		@include responsive('tablet') {
 			display: none;
 		}
 	}
+
+	&__menu-icon-svg {
+		pointer-events: none;
+	}
+}
+
+.menu {
+	&__slider {
+		position: fixed;
+		top: 0;
+		right: 0;
+		height: 100%;
+		width: 0;
+		background-color: var(--color-text-secondary);
+		z-index: 2000;
+		padding-top: 3.4rem;
+		border-radius: 5px;
+		overflow-x: hidden;
+		transition: 0.5s;
+	}
+
+	&__slider-item-container{
+		display: flex;
+		align-self: center;
+		justify-content: center;
+		flex-direction: column;
+		margin-top: 5rem;
+	}
+
+	&__slider-item-container{
+		display: flex;
+		align-self: center;
+		justify-content: center;
+		flex-direction: column;
+		margin-top: 5rem;
+	}
+
+	&__slider-item {
+		display: block;
+		text-decoration: none;
+		text-align: center;
+		margin: 2rem 2rem;
+		font-size: 2.5rem;
+		transition: 0.1s;
+
+		&:visited {
+			color: var(--color-text-primary);
+		}
+
+		&:hover {
+			color: var(--color-background-darker);
+			transition: 0.3s;
+		}
+	}
+
+	&__button-close {
+		display: flex;
+		justify-content: center;
+		position: absolute;
+		top: 1rem;
+		left: 1rem;
+		cursor: pointer;
+		width: 3rem;
+		height: 3rem;
+	}
+
+	&__button-svg {
+		pointer-events: none;
+	}
+}
+
+@include responsive('tablet') {
```

Comment:
this could be moved to be under &__menu__slider in a similar way to the other instances

Reply:
Done